grails-app/conf/Config.groovy

```diff
@@ -277,7 +277,7 @@ grails.gorm.default.mapping = {
 
 detection {
     extract {
-        limit = 50000
+        fetchSize = 1000
```

Comment:
Do we know how this performs for a large CSV download, e.g. how long does it take to download 1M records?  Ok if it's not been tested, but we should check this on systest.